python/test/ngraph/util.py

```diff
@@ -45,25 +48,24 @@ def run_op_node(input_data, op_fun, *args):
     comp_args = []
     op_fun_args = []
     comp_inputs = []
-    for idx, data in enumerate(input_data):
-        if np.isscalar(data):
-            op_fun_args.append(ng.constant(data, _get_numpy_dtype(data)))
-        else:
-            node = ng.parameter(data.shape, name=ascii_uppercase[idx], dtype=data.dtype)
-            op_fun_args.append(node)
-            comp_args.append(node)
-            comp_inputs.append(data)
+    for data in input_data:
+        op_fun_args.append(ng.constant(data, _get_numpy_dtype(data)))
     op_fun_args.extend(args)
     node = op_fun(*op_fun_args)
     computation = runtime.computation(node, *comp_args)
     return computation(*comp_inputs)
 
 
 def run_op_numeric_data(input_data, op_fun, *args):
+    # type: (NumericData, Callable, *Any) -> List[NumericData]
     """Run computation on node performing `op_fun`.
 
     `op_fun` has to accept a scalar or an array.
 
+    This function passess input data AS IS. This mean that in case they're a scalar (integral,
+    or floating point value) or a NumPy's ndarray object they will be automatically converted
+    to nGraph's Constant Nodes.
```

Comment:
Please add a note what happens otherwise - with an example.
Please also explain why this function is needed.

Reply:
Ad 2) This function is needed for unit testing purposes only - as you may see it is used only in our unit tests. 
Ad 1) Actually there is no _otherwise_ - it is used and defined to run on numeric data only.